docs/user/02-configuration.md

```diff
@@ -73,6 +73,22 @@ Use the following sample CRs as guidance. Each can be applied immediately when y
 | **publisherService**                                 | string     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | **specHash** (required)                              | integer    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-| **state** (required)                                 | string     | Can have one of the following values: Ready, Error, Processing, Warning. Ready state is set when all the resources are deployed successfully and backend is connected. It gets Warning state in case backend is not specified and NATS module is not installed or EventMesh secret is missing in the cluster. Error state is set when there is an error. Processing state is set if recources are being created or changed. |
+| **state** (required)                                 | string     | Can have one of the following values: `Ready`, `Error`, `Processing`, `Warning`. `Ready` state is set when all the resources are deployed successfully and backend is connected. It gets `Warning` state in case backend is not specified, NATS module is not installed or EventMesh secret is missing in the cluster. `Error` state is set when there is an error. `Processing` state is set if recources are being created or changed. See the next section for more details. |
 
 <!-- TABLE-END -->
+
+<h3>More Details about Eventing CR state</h3>
```

Comment:
```suggestion
<h3>More Details About Eventing CR State</h3>
```
All [headlines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#title-case) must be in title case.